CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v1.12.0
+* Enable overriding proxy buffer values. Defaults to `proxy_buffer_size 16k` and `proxy_buffers 4 16k`
```

Comment:
2 lines duplicate with v1.11.3

Reply:
Yeh, I did not mean to release the `proxy_buffer` changes as a patch version. I am hoping I could include the changes from this and release them together as `1.12.0`. We then can deprecate `1.11.4` if that's an option. Although changes in `1.11.4` is not a breaking change but it is a new functionality and it makes sense to release them as a minor release.

Reply:
It's actually `1.11.4`. Is it ok if I remove the change notes for that release or mark it as depcrecated or donot use ? We can use `1.12.0` as the official release which supports large requests and responses.